layouts/partials/footer.html

```diff
@@ -28,7 +28,7 @@
           </script>
         {{ end }}
 
-        <script src="{{ .Site.BaseURL }}js/main-dd.js"></script>
+        <script src="{{ .Site.BaseURL }}js/{{ (index .Site.Data.manifests.js "main-dd.js") }}"></script>
```

Comment:
static assets are loaded from a manifest which will map friendly name to hashed name